Code/Tools/FBuild/FBuildCore/ExeDrivers/Compiler/CompilerDriverBase.cpp

```diff
@@ -134,6 +134,13 @@ void CompilerDriverBase::Init( const ObjectNode * objectNode,
 {
 }
 
+// AddPreliminaryArgs_Common
+//------------------------------------------------------------------------------
+/*virtual*/ void CompilerDriverBase::AddPreliminaryArgs( bool /*isLocal*/,
+                                                         Args & /*outFullArgs*/ ) const
+{
```

Comment:
Generally FASTBuild tries not to automatically add or modify command line args, except where strictly necessary.

Args that are always needed usually are specified in the bff file by the user.